### DISCUSSION TITLE
Not working in Macos Ventura

### DISCUSSION THREAD

**Discussion by Trukmeister** [debauchee/barrier#1899] [General]
Barrier will not work after installing the latest Macos Ventura.  After starting Barrier macos requires access to "accessibility features"  Even after enabling access (and disabling and re-enabling many times) Barrier will not run.  The OS stops it no mater what,  Any ideas or is an update needed? Thanks

**Comment by aelie** [comment]
Same here, Barrier refuses to start due to bad recognition of the accessibility features setting.
Since Ventura 13.2.1, was working fine until now.

Thanks for giving it a look!

**Comment by Trukmeister** [comment]
I resolved my problem by removing the Accessibility where Barrier is.  In the new OS it's a little different to remove items from Accessibility.  Once I figured that out and reinstalled Barrier it worked fine.

**Reply by jakovos1** [reply]
Wow how did you do that ?

**Reply by Trukmeister** [reply]
I am not sure of the exact spot that you need to do this but when you install Barrier the install process brings up the setting that you need to add.  However since (for me) Barrier was previously installed the Barrier Accessibility setting was already turned on.  Turning it off and back on again did not correct the problem.  The delete (for the setting) option was not available either - not sure why. Anyway after un/installing several times the + and - buttons were there.  So in the Accessibility section that the Barrier install process brings up you need to select the Barrier entry and hit the minus button while it is selected and remove barrier, uninstall Barrier yet again and then install it yet again and this time activate the Barrier Accessibility button and it should work.  This screenshot included may not be the exact Accessibility section you need but it gives you the idea.
<img width="703" alt="Apple-settings" src="https://user-images.githubusercontent.com/76884571/228563944-2d2e8e2f-0ea0-4c04-baa3-361079ddd3f6.png">

**Reply by fadersolo** [reply]
I just updated to Ventura and ran into the same issue. Thanks the helpful post, this solved it for me!

**Comment by AnimalFromTheZoo** [comment]
Currently on M2 , Mac OS 13.3, Needed to control my Windows 10 PC it would not work for 2 days till i tried to do this command,
I reinstalled using command in the Terminal after installing [Homebrew ](https://brew.sh/) : brew reinstall --cask barrier, 
Pressed the MINUS BUTTON on Accessibility Panel and relaunched and gave the Permission on Accessibility Panel again the SSL certificate worked! Hope this helps another soul that is trying to get it to work! 🥇 Good Luck Fellow Barrier Friends!

**Comment by man-chan** [comment]
Just tired this on my M1 with Ventura, it works. Thanks.

**Comment by alrucks** [comment]
@Trukmeister solution works like a charm! I did not have to uninstall, just removing Barrier from the Accessibility with the minus sign, and then starting Barrier again, opening System Settings and giving it permission again by turning on its line on Accessibility made the trick!

**Comment by fortegaseguel** [comment]
Sonoma doesn't work for me. The server log don't register any connection and the server's log shows Time out. I try the suggested solutions but doesn't work

**Comment by klepl** [comment]
@fortegaseguel Disabling Barrier SSL worked for me (Sonoma). See: https://www.reddit.com/r/MacOS/comments/11x9ir2/any_one_tried_to_run_barrier_kvm_application_on/
